cmake/Modules/FindMKL.cmake

```diff
@@ -17,6 +17,9 @@
 # ---[ Options
 include(CMakeDependentOption)
 
+option(MKL_USE_IDEEP "Use IDEEP interface in MKL BLAS" ON)
```

Comment:
All options should be in top-level CMakeLists. BTW, any reason to not call it `USE_MKL_IDEEP`?